project.clj

```diff
@@ -1,4 +1,5 @@
 (def ps-version "2.7.3-SNAPSHOT")
+(def tk-jetty9-version "1.7.1-SNAPSHOT")
```

Comment:
This will need to be replaced with a new tk-jetty9 release when one is available.